.circleci/config.yml

```diff
@@ -8,11 +8,15 @@ jobs:
       - run:
           name: Update repos.json
           command: |
+            python -m venv venv
+            . venv/bin/activate
+            pip install -e .
             make repos.json
       - save_cache:
           key: deps1-{{ .Branch }}
```

Comment:
Will this cache key still ensure that changes to the repo will result in reinstallation?

Reply:
No, the cache key is a relic I still had in the config and it doesn't make any sense at all in this setup. That's why I updated it in 132d890.